site/docs/4.2/assets/scss/_component-examples.scss

```diff
@@ -243,7 +243,7 @@
 }
 
 // List groups
-.bd-example > .list-group {
+.bd-example > .list-group:not([class*="horizontal"]) {
```

Comment:
This is a no-go for me, attribute selectors like these are way less performant than the universal selector and are not strict enough because it could potentially target other classes that contain horizontal

Reply:
Usually I'm in the same boat, but this being a docs-only thing, I was fine with it. I can easily rewrite it though to override the previous selector.

Reply:
Ow, didn't notice that. In that case: nevermind